package/MDAnalysis/lib/util.py

```diff
@@ -1620,6 +1622,8 @@ def unique_rows(arr, return_index=False):
     --------
     Remove dupicate rows from an array:
 
+    >>> import numpy as np
+    >>> from MDAnalysis.lib.util import unique_rows
     >>> a = np.array([[0, 1], [1, 2], [1, 2], [0, 1], [2, 3]])
     >>> b = unique_rows(a)
     >>> b
```

Comment:
The result following this one (`array([[0, 1], [1, 2], [2, 3]])`) fails because it is not formatted properly.

```text
Failed example:
    b
Expected:
    array([[0, 1], [1, 2], [2, 3]])
Got:
    array([[0, 1],
           [1, 2],
           [2, 3]])
```